_includes/manuals/1.16/1.2_release_notes.md

```diff
@@ -3,9 +3,16 @@
 ### Headline features
 
 #### Netgroup LDAP authentication
+Netgroup grouping is alternative to posix usergroups. Foreman can now be configured to pull users from external groups using either posix or Netgroup naming standards. Although this is an older standard, it can be useful for integration with [FreeIPA](https://www.freeipa.org/) and other servers. ([#16112](http://projects.theforeman.org/issues/16112))
```

Comment:
maybe it's worth of mentioning that it's supported for FreeIPA and OpenLDAP only, since there was no easy way for AD

Reply:
@areas I updated the text with OpenLDAP. I did not mention lack of support for other servers. Let me know if you think the new text is sufficient.

Reply:
I don't see OpenLDAP mentioned but I think the text is OK, thanks

Reply:
Oh nvm, I see it now :-)